### PR TITLE
Add the ability to have a ReadOnly opener.

### PR DESCRIPTION
**Pull request by jwessel**
For security purposes the end user may not want to allow any kind of voice command which can operate an opener.

Signed-off-by: Jason Wessel <jason.wessel@windriver.com>

### CODE REVIEW

**Comment by hjdhjd** [comment]
Appreciate the contribution, but the better way to ensure a read-only (or even time-specific availability) is really through the myQ ecosystem itself. Create a second user account, and assign it permissions (or not) to address the need. It's got a lot more flexibility built into it than implementing something like this will.

I had initially considered providing this functionality, but I considered it duplicative and inferior to ultimately what myQ was able to offer natively through their guest / shared access capabilities, and I'd rather have users contend with managing permissions to their garage door openers from one place rather than two.

I'm open to being convinced that there's a good use case to add additional capability in homebridge-myq, but I tend to not add functionality that's already available (in this case in an arguably better form) and accessible through the plugin.

TL;DR: my view is that the right place for this is in the myQ app through the use of guest / shared access and managing permissions there, rather than creating an additional access layer through homebridge-myq that's superfluous.

Best of luck and thanks again!

**Comment by hjdhjd** [comment]
Thanks for the feedback. As time allows, I’ll take a closer look at how myQ has evolved their guest access capabilities.

The issue isn’t the code. The code is trivial. The issue is capability that I want to enable with this plugin and I have a high bar for what makes it in.

Thanks again.

**Comment by hjdhjd** [comment]
Added this capability in v2.9.0. Thanks for the dialog!
